grails-app/i18n/messages.properties

```diff
@@ -54,3 +54,142 @@ typeMismatch.java.lang.Short=Property {0} must be a valid number
+
+howtostep1.select = Select a KBART file*  
+howtostep1.select.sup = * with zdb_id if applicable
+howtostep1.format = Specify the file format
+howtostep1.mediatype = Specify the media type 
+howtostep1.upload = Upload the KBART file
+
+howtostep2.packagename = Insert an (existing) <strong>package name</strong> for the GOKb package
+howtostep2.curatorygroup = Insert the name of your <strong>curatory group</strong>.
+howtostep2.ebooks = If you selected 'e-books' as media type, you have to select an existing identifier namespace, so that the provider's proprietary title id can be matched or created.
+howtostep2.titlemismatch = <em>If the chosen package name doesn't match an existing GOKb package name, a new package will be created.</em>
+howtostep2.platformmismatch <em>Without a valid platform no nominal platform will be linked to the package and the tipps.</em>
+
+howtostep3.sources = Select sources for data enrichment.
+howtostep3.start = Start data enrichment.
+
+howtostep4.save = Save the result files (titles and package) and debug files on your hard disk.
+howtostep4.send = Send the result to a connected GOKb instance.
+howtostep4.sequence = <em>Send titles first and package last.</em> 
+howtostep4.nomanualcorrection = <em>Currently it is not possible to revise the results manually before sending.</em>
+
+howtostep4hint.hint = Hint
+howtostep4hint.deleted = Files transmitted to this server will be deleted after some time.
+
+listDocuments.key.filename = File
+listDocuments.key.title = Title
+listDocuments.key.mediatype = Source Data Type
+listDocuments.key.packagename = Package Name
+listDocuments.key.isil = Package ISIL (ZDB)
+listDocuments.key.namespace = Namespace for provider's proprietary title identifier (title_id)
+listDocuments.key.platformname = Platform name
+listDocuments.key.platformurl = Platform URL
+listDocuments.key.platform = platform
+listDocuments.key.provider = provider
+listDocuments.key.sources = Sources
+listDocuments.key.key = Key
+listDocuments.mediatype.journal = E-Journals
+listDocuments.mediatype.ebook = E-Books
+listDocuments.mediatype.database = Database
+listDocuments.button.deletefile = Drop file
+listDocuments.button.correctfile = Change file
+listDocuments.button.processfile = Process file
+listDocuments.button.stopprocessingfile = Stop processing
+listDocuments.button.downloadpackagefile = Download package file
+listDocuments.button.downloadtitlesfile = Download titles file
+listDocuments.button.package = Send <em>package</em> to GOKb
+listDocuments.button.titles = Send <em>titles</em> to GOKb
+listDocuments.button.cancel = Cancel
+listDocuments.button.downloadDebugFile = Download debug file
+listDocuments.button.downloadRawFile = Download RAW file
+listDocuments.button.sendPackageFile = Send package to GOKb
+listDocuments.button.sendTitlesFile = Send titles to GOKb
+listDocuments.button.send = Send
+listDocuments.gokb.credentials = Please enter your GOKb credentials
+listDocuments.gokb.username = Username
```

Comment:
@darupp Some German phrases left here... lines 169, 170, 172

Reply:
Thx, I added those and commited again.